app/src/main/java/com/github/polypoly/app/ui/game/HeadsUpDisplay.kt

```diff
@@ -284,18 +294,20 @@ fun HudGameMenu() {
  * A button that is used in the HUD
  */
 @Composable
-fun HudButton(name: String, onClick: () -> Unit, icon_id: Int, description: String) {
+fun HudButton(name: String, onClick: () -> Unit, icon: ImageVector, description: String) {
```

Comment:
`HudButton` Can be replace by `IconRoundButton` in `UIElements` which is almost the same composable, and which is already used in different files of the project

Reply:
I tried IconRoundButton, but I prefer this custom shaped HudButton implementation for this specific usage.